pkg/apis/ceph.rook.io/v1/object.go

```diff
@@ -27,3 +37,34 @@ func (s *ObjectStoreSpec) IsExternal() bool {
 func (s *ObjectRealmSpec) IsPullRealm() bool {
 	return s.Pull.Endpoint != ""
 }
+
+func (o *CephObjectStore) ValidateCreate() error {
```

Comment:
It would be nice to have a common code for the validation, we need a uniq place, see:

https://github.com/rook/rook/blob/a134d1df6ed2a72507e6ba6338f2c414b9987b5a/pkg/operator/ceph/object/rgw.go#L282-L308

Reply:
Done.